pkg/api/datasources_test.go

```diff
@@ -481,18 +481,22 @@ func TestAPI_datasources_AccessControl(t *testing.T) {
 	}
 }
 
-// TeamHTTPHeaderValueRegexMatch returns a regex that can be used to check
-func TestTeamHTTPHeaderValueRegexMatch(t *testing.T) {
+func TestValidateLBACHeader(t *testing.T) {
 	testcases := []struct {
 		desc            string
 		teamHeaderValue string
 		want            bool
 	}{
 		{
-			desc:            "Should be valid regex match for team headervalue",
+			desc:            "Should allow valid header",
 			teamHeaderValue: `1234:{ name!="value",foo!~"bar" }`,
 			want:            true,
 		},
+		{
+			desc:            "Should allow valid selector",
+			teamHeaderValue: `1234:{ name!="value",foo!~"bar/baz.foo" }`,
```

Comment:
nice!

Reply:
this is a valid selector?
"bar/baz.foo"

Reply:
yes. Ie file name "/foo/bar.log"